yaptide/batch/watcher.py

```diff
@@ -2,13 +2,19 @@
 import re
 import signal
 import time
+import json
+import ssl
 from pathlib import Path
+from urllib import request
+import logging
+from datetime import datetime
 
 
 RUN_MATCH = r"\bPrimary particle no.\s*\d*\s*ETR:\s*\d*\s*hour.*\d*\s*minute.*\d*\s*second.*\b"
 COMPLETE_MATCH = r"\bRun time:\s*\d*\s*hour.*\d*\s*minute.*\d*\s*second.*\b"
 REQUESTED_MATCH = r"\bRequested number of primaries NSTAT"
 TIMEOUT_MATCH = r"\bTimeout occured"
+HARDCODED_BACKEND_URL = "https://yap-dev.c3.plgrid.pl:8443"
```

Comment:
Can you send that URL when submitting job ? On the other hand backend itself could retrieve its own URL from an environmental variable. I can extend the deploy script to handle it.

Reply:
Of course I can. This was temporary HARDCODED solution. When you change deploy I can change it here

Reply:
Start changing it here, I will adjust deploy later. You can always test it by `MYNEWENV=myblablavalu docker compose up` and adding a line like that:
https://github.com/yaptide/yaptide/blob/master/docker-compose.yml#L18
and like that:
https://github.com/yaptide/yaptide/blob/28344fbd0ea4aa79fa5fa637cbdafa3b95f1ebe1/Dockerfile-flask#L9

I'd call this name BACKEND_EXTERNAL_URL